tsdb/shard_test.go

```diff
@@ -48,9 +48,6 @@ func TestShardWriteAndIndex(t *testing.T) {
 	}
 
 	validateIndex := func() {
-		if !reflect.DeepEqual(index.Names(), []string{"cpu"}) {
```

Comment:
It already looks for the measurement by name in the `measurements` map on L59.  `Names()` is never used by actual code.  Just this test code.